src/ant/README.md

```diff
@@ -0,0 +1,45 @@
+# Antenna Rule Checker
+
+This tool checks antenna violations based on an input LEF and DEF file loaded in opendb, a report
+will be generated to indicate violated nets. APIs are provided to help fix antenna violation during the diode insertion flow in globalroute:
+
+## Antenna Check Commands
+
+---
+
+ - `load_antenna_rules`: import antenna rules to ARC, must be called before other commands
```

Comment:
load_antenna_rules is not a command, it an internal tcl function called by check_antennas so it does not belong here

Reply:
removed